ChangeLog.md

```diff
@@ -16,13 +16,24 @@
 * Fixed crash when player has manual control of arrow towers during siege
 * Fixed crash on attempt to attack with Magma Elementals with Erdamon as hero
 * Fixed crash on attempt to access removed Quest Guard
+* Fixed crash on moving through whirlpool when hero has no troops other than commander
+* Fixed possible freeze when moving hero over events that give enough experience to cause a level-up
+* Fixed possible crash on movement of double-wide creatures next to gates during siege
 
 ### Multiplayer
 * Implemented new lobby, available in game with persistent accounts and chat
 * Removed old lobby previously available in launcher
 * Fixed potential crash that could occur if two players act at the very same time
+* Game will no longer pause due to network lag after every tile when instant movement speed is selected in multiplayer
 
 ### Interface
+* Implemented configurable keyboard shortcuts, editable in file config/shortcutsConfig.json
+* Fixed broken keyboard shortcuts in main menu
+* Added option to skip confirmation dialogs when entering owned dwellings or refugee camp
```

Comment:
Maybe it's good idea to reword this to indicate it is already existing "enable UI enhancements" option that has this built-in